components/forms/input-fields/input-field.config.yml

```diff
@@ -3,5 +3,6 @@ notes: 'Overview of input-fields'
 context:
   placeholder: "Enter your e-mail*"
   message: "We'll never share your email with anyone else."
-  errorMessage: "You have entered and invalid email address. Please try again."
-  successMessage: "Looks good!"
\ No newline at end of file
+  errorMessage: "You have entered and invalid email address. Please try again. You have entered and invalid email address. Please try again. You have entered and invalid email address. Please try again."
```

Comment:
Does the errorMessage need to be repeated 3 times?

Reply:
i forgot to remove the extra text:) ill trim it down